### ISSUE TITLE
Handling aliased DOIs at CrossRef API: Resource not found?

### ISSUE THREAD

**Issue by ioverka** [CrossRef/rest-api-doc#110]
In rare cases, multiple DOIs have been assigned to the same content accidentially. In these cases we ask the corresponding publisher to alias both DOIs to each other, see http://help.crossref.org/forcing-prime_alias. 

Here is an example for an aliased DOI:
- http://doi.org/10.1007/s11956-008-1009-1 = primary DOI, redirects to location: http://link.springer.com/10.1007/s11956-008-1009-1
- http://doi.org/10.3103/S009545270801009X = aliased DOI, redirects to location: http://doi.org/10.1007/s11956-008-1009-1

At the CrossRef API, only the primary DOI returns a record:
- http://api.crossref.org/works/10.1007/s11956-008-1009-1
- http://api.crossref.org/works/10.3103/S009545270801009X = Resource not found

Wouldn't it be more appropriate if the request for an aliased DOI would return the record for the primary DOI - or at least point to the primary DOI?

thanks, inga

**Comment by kjw** [comment]
Yes it would. At the moment data on aliased DOIs doesn't get streamed to the component indexing data for the REST API. I've got this on my internal todo list but getting that data requires changes to systems throughout Crossref. Stay tuned.

**Comment by kjw** [comment]
@ckoscher : Reminder that it would be useful to somehow see aliased DOI information, via deposit feed API, via OAI-PMH?

**Comment by ckoscher** [comment]
The unixsd for a DOI does point to a prime(crm-item) if that doi is an alias to some other doi.

http://doi.crossref.org/search/doi?pid=ckoscher@crossref.org&format=unixsd&citeid=35424

This data would be in the deposit feed to the indexer. Normally an aliased DOI can not be updated so its updated-date would never increment thus forcing it to be pushed. When a DOI first gets aliased its updated-date is changed so that event would cause it to get indexed, so we have that covered. So, I see 2 steps needed.

1) make sure the cayenne indexer processes the < crm-item name="prime-doi" ...>
2) figure out how to push all existing aliased DOIs  via the feed

**Comment by ioverka** [comment]
Thanks, it's good to know that this issues is on your agenda already. Unfortunately, DOI aliasing doesn't seem to be very popular among publishers...

**Comment by gbilder** [comment]
Is this now handled correctly by API as well?

**Comment by ioverka** [comment]
Hm, both DOIs now resolve via the API - which is an improvement ! - but it's left to the experts to figure out that one of them is aliased to the other, i.e. it's only in the `<crm-item name="prime-doi"` element in unixds: http://api.crossref.org/works/10.3103/S009545270801009X/transform/application/vnd.crossref.unixsd+xml

Any chance to make this more obvious?

In addition: May I suggest that the CrossRef search should covers primary DOIs only? http://search.crossref.org/?q=Interaction+of+RNA-containing+bacteriophages+with+host+cell%3A+MS2-induced+mutants+of+E.+coli+and+the+occurrence+of+DNA-containing+derivatives+of+the+bacteriophage+MS2

**Comment by ioverka** [comment]
@gbilder ... is adding a "prime-doi" really the same than a DOI alias?

It looks like this setup still includes two complete DOI records - which probably both need to be updated independently. Is that really how it's supposed to be? "Alias" sounds like an alternative identifier for the same thing.

**Comment by rordi** [comment]
For others stumbling across this issue with DOI aliases. Handle system includes HS_ALIAS info for aliased DOI numbers: e.g. http://hdl.handle.net/api/handles/10.4081/idr.2009.e4
